enterprise/app/org/org_user_lists.tsx

```diff
@@ -0,0 +1,284 @@
+import React from "react";
+import { User } from "../../../app/auth/auth_service";
+import Banner from "../../../app/components/banner/banner";
+import Button, { OutlinedButton } from "../../../app/components/button/button";
+import Dialog, {
+  DialogBody,
+  DialogFooter,
+  DialogFooterButtons,
+  DialogHeader,
+  DialogTitle,
+} from "../../../app/components/dialog/dialog";
+import TextInput from "../../../app/components/input/input";
+import Modal from "../../../app/components/modal/modal";
+import Spinner from "../../../app/components/spinner/spinner";
+import errorService from "../../../app/errors/error_service";
+import router from "../../../app/router/router";
+import rpcService from "../../../app/service/rpc_service";
+import { grp } from "../../../proto/group_ts_proto";
+import { user_list } from "../../../proto/user_list_ts_proto";
+import OrgUserListComponent from "./org_user_list";
+import UserListComponent, { UserListUser } from "./user_list";
+
+export type OrgMembersProps = {
+  user: User;
+};
+
+type State = {
+  loading?: boolean;
+  response?: user_list.GetUserListsResponse;
+
+  editingList?: boolean;
+
+  // state related to the create user list modal
+  isCreateUserListModalVisible?: boolean;
+  createUserListRequest?: user_list.CreateUserListRequest;
+  isCreateUserListRequestPending?: boolean;
+
+  // state related to the delete user list modal
+  isDeleteUserListModalVisible?: boolean;
+  userListToBeDeleted?: user_list.UserList;
+  isDeleteUserListRequestPending?: boolean;
+};
+
+const DEFAULT_ROLE = grp.Group.Role.DEVELOPER_ROLE;
+
+export default class OrgUserListsComponent extends React.Component<OrgMembersProps, State> {
+  state: State = {
+    loading: true,
+  };
+
+  private removeButtonRef = React.createRef<HTMLButtonElement>();
+
+  componentDidMount() {
+    this.fetch();
+  }
+
+  componentDidUpdate(prevProps: OrgMembersProps, prevState: State) {
+    if (!prevState.isDeleteUserListModalVisible && this.state.isDeleteUserListModalVisible) {
+      setTimeout(() => {
+        this.removeButtonRef.current?.focus();
+      }, 0);
+    }
+
+    // Refresh data when navigating back from an individual user list.
+    if (this.state.editingList && prevState.editingList) {
```

Comment:
this is gross but i'm not sure how to fix this. `onClickEditUserList` navigates to the "edit user list" page. when navigating back from that page, we need to refresh the user lists since that page could have modified them.